virtme/guest/virtme-sshd-script

```diff
@@ -24,18 +24,37 @@ SSH_AUTH_KEYS="${SSH_HOME}/.ssh/authorized_keys"
 if [ "$(stat -f -c "%t" "${SSH_AUTH_KEYS}")" = "${OVERLAYFS}" ]; then
     cat "${SSH_HOME}"/.ssh/id_*.pub >> "${SSH_AUTH_KEYS}" 2>/dev/null
     chown "${virtme_ssh_user}" "${SSH_AUTH_KEYS}" 2>/dev/null
+    chmod 600 "${SSH_HOME}/.ssh/authorized_keys" 2>/dev/null
 fi
 
 # Generate ssh host keys (if they don't exist already).
 CACHE_DIR=${SSH_HOME}/.cache/virtme-ng/.ssh
 mkdir -p "${CACHE_DIR}/etc/ssh"
 ssh-keygen -A -f "${CACHE_DIR}"
+
+# Generate a minimal sshd config.
+SSH_CONFIG=/etc/ssh/sshd_config
+if [ "$(stat -f -c "%t" "${SSH_CONFIG}")" = "${OVERLAYFS}" ]; then
+    ssh_dir=$(dirname "${SSH_CONFIG}")
+    mkdir -p "${ssh_dir}"
+    cat << EOF > "${SSH_CONFIG}"
+# This file is automatically generated by virtme-ng.
+Port 22
+PermitRootLogin yes
+AuthorizedKeysFile .ssh/authorized_keys
+PubkeyAuthentication yes
+UsePAM yes
```

Comment:
I wonder if this option can be removed if there are no passwords (same for the two above). I guess all Linux distributions have PAM support.

Reply:
I think we need `UsePAM yes` for the pubkey authentication, at least in my case if I remove that it's asking me the password and it doesn't work...

Reply:
> I think we need `UsePAM yes` for the pubkey authentication, at least in my case if I remove that it's asking me the password and it doesn't work...

Yes indeed, we need it for the moment. But maybe we don't need it with https://github.com/arighi/virtme-ng-init/pull/16 ?